examples/networking/floatingip_associate/main.tf

```diff
@@ -0,0 +1,7 @@
+resource "vkcs_networking_floatingip_associate" "floatingip-associate" {
+  floating_ip = vkcs_networking_floatingip.floatingip.address
+  port_id     = vkcs_networking_port.port.id
+  # ensure the router interface is up
+  # ensure that router is connected to external network
```

Comment:
We do not ensure this here. If you want to mention this - change the resource description

Reply:
changed